test/Engine/ProtoTestFx/TestFrameWork.cs

```diff
@@ -397,6 +402,32 @@ public virtual ExecutionMirror RunScriptSource(string sourceCode, string errorst
         {
             sourceCode = sourceCode.Insert(0, "import(\"DesignScriptBuiltin.dll\");");
 
+            if (testMSILExecution)
+            {
+                Console.WriteLine($"!!!!test {NUnit.Framework.TestContext.CurrentContext.Test.Name} being run with MSIL compiler!!!!");
+                //TODO_MSIL: remove the dependency on the old VM by implementing
+                //necesary Emit functions(ex mitFunctionDefinition and EmitImportStatements and all the preloading logic)
+                //MSIL_TODO add list of libraries to load here for tests.
+                //It seems we have to load SOME library successfully or DSExecutable is null.
+                using (var liveRunner = new ProtoScript.Runners.LiveRunner())
+                {
+                    liveRunner.ResetVMAndResyncGraph(new string[] { "DesignScriptBuiltin.dll" });
```

Comment:
Why do you need to reset the VM here?

Reply:
I was following the pattern @pinzart90 used here:
https://github.com/DynamoDS/Dynamo/blob/emitMSIL/test/CodeGenILTests/RangeMicroTests.cs#L34

If I did not call this method with some valid library then `DSExecutable` seemed to always be null.

Reply:
but can't you use the same instance of the old VM that is already initialized? I mean isn't `liveRunner.runtimeCore` and the rest of the stuff like `DSExecutable` etc. already initialized in these tests?

Reply:
Not really, the tests start a testcore and then create a runtimecore sometime later, it's all bound up in the existing execute method and I didn't see the value in refactoring this - it's not obvious to me we're going to keep all this legacy infrastructure...

Reply:
I can take a look at reusing it if you think it's worth it? I was hoping we could run these tests with only import statements at some point.

Reply:
Ok, thanks.

Reply:
> I can take a look at reusing it if you think it's worth it? I was hoping we could run these tests with only import statements at some point.

No, I don't think it's worth it if we are going to remove the dependency on the old runtime core at some point later.